deployability/modules/workflow_engine/examples/agent/vagrant/test-agent-complete-macOS.yaml

```diff
@@ -0,0 +1,113 @@
+version: 0.1
```

Comment:
The macOS workflow file is duplicated:
- deployability/modules/workflow_engine/examples/agent/vagrant/test-agent-complete-macOS.yaml
- deployability/modules/workflow_engine/examples/agent/vagrant/test-agent-complete-macOs.yaml

Please remove one of them.

Reply:
Done https://github.com/wazuh/wazuh-qa/pull/5310/commits/d3a2c09d2504cfc5e26fc9be6939934cae56ddbe